### PR TITLE
ALS-6735: Transition from explore to discover page

### PR DESCRIPTION
**Pull request by ramari16**
Still todo:
* Handle stigmatizing variables

### CODE REVIEW

File: src/lib/components/explorer/export/ExportStepper.svelte [hunk 1]
```diff
@@ -116,8 +116,11 @@
 
   async function createNamedDataset() {
     try {
-      const datasetName = encodeURIComponent(datasetNameInput);
```
Comment (ramari16) [inline src/lib/components/explorer/export/ExportStepper.svelte:119]: We were double-encoding spaces which was causing chaos

File: src/lib/stores/Filter.ts [hunk 2]
```diff
@@ -7,6 +7,10 @@ export const totalParticipants: Writable<number | string> = writable(0);
 export const hasGenomicFilter: Readable<boolean> = derived(filters, ($f) =>
   $f.find((filter) => filter.filterType === 'genomic') ? true : false,
 );
+export const hasStigmatizedFilter: Readable<boolean> = derived(filters, ($f) =>
```
Comment (ramari16) [inline src/lib/stores/Filter.ts:10]: This is temporary

File: src/routes/(picsure)/(public)/discover/+page.svelte [hunk 3]
```diff
@@ -2,13 +2,43 @@
...
 <Content full>
-  <Explorer tourConfig={openTour} />
+  {#if $hasGenomicFilter || $hasUnallowedFilter}
+    <section id="discover-error-container" class="flex gap-9">
+      <ErrorAlert
+        title="Your selected filters contain stigmatizing variables and/or genomic filters, which are not supported with Discover"
+      >
+        <p>Would you like to remove the invalid filters or go back to explore?</p>
+        <div class="dark">
+          <button class="btn variant-filled" on:click={() => resetQuery()}
```
Comment (JamesPeck) [inline src/routes/(picsure)/(public)/discover/+page.svelte:32]: variant-filled-primary?

File: src/lib/components/explorer/export/ExportStepper.svelte (hunk 1 above)
Comment (JamesPeck) [reply src/lib/components/explorer/export/ExportStepper.svelte:119]: where is the second time?

File: src/lib/services/datasets.ts [hunk 4]
```diff
@@ -5,10 +5,6 @@ export async function createDatasetName(queryId: string, name: string): Promise<
   if (name === '' && name.trim() === '') {
     throw 'Please input a Dataset ID name';
   }
```
Comment (JamesPeck) [inline src/lib/services/datasets.ts:7]: I think the other validation should get moved back here or this should get moved as well. Seeing that this datasets.ts could be used elsewhere I think it should be here. (We have talked above allowing them to save the dataset without packaging the data)

**Pull request merged**